setup1-contributor/src/commands/contribute.rs

```diff
@@ -750,21 +759,69 @@ impl Contribute {
             .error_for_status()?;
         Ok(())
     }
+
+    async fn prompt_eth_address<R: Rng + CryptoRng>(&self, auth_rng: &mut R) -> Result<()> {
+        println!(
+            "As a token of our appreciation, we would like to send an NFT to all participants, which shows you've participated in the Aleo setup ceremony. If you would like to receive such an NFT, please enter your ETH address! If not, just hit enter."
```

Comment:
Thank you for participating in Aleo Setup. As a token of appreciation, we would like to send you a commemorative NFT. This NFT is procedurally generated, but represents your unique contribution. We hope it serves as a reminder of the important role that YOU played in bringing Aleo to life.